pkg/agent/openflow/client_windows.go

```diff
@@ -51,3 +57,20 @@ func (c *client) UninstallLoadBalancerServiceFromOutsideFlows(svcIP net.IP, svcP
 	cacheKey := fmt.Sprintf("L%s%s%x", svcIP, protocol, svcPort)
 	return c.deleteFlows(c.serviceFlowCache, cacheKey)
 }
+
+func (c *client) InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16) error {
+	c.replayMutex.RLock()
+	defer c.replayMutex.RUnlock()
+	var flows []binding.Flow
+	flows = append(flows, c.serviceLBFlow(groupID, svcIP, svcPort, protocol, affinityTimeout != 0))
+	if affinityTimeout != 0 {
+		flows = append(flows, c.serviceLearnFlow(groupID, svcIP, svcPort, protocol, affinityTimeout))
+	}
+	if c.enableProxy {
+		flows = append(flows, c.arpResponderFlow(route.VirtualGWIP, virtualSvcMAC, priorityHigh, cookie.Service))
```

Comment:
Just curious, are these flows (arpRespond flows and l3FwdFlowClusterIPServices) only used on Windows? I mean does Linux need them?

Reply:
I think Linux needs them.

Reply:
This can be removed. I added a static neighbor instead of this.

Reply:
@hongliangl does it work for all platforms?

Reply:
I don't test it in Windows, I think you can try it manually in Windows.

Reply:
Please add comments, like:

> ClusterIP Service requests will be sent to OVS bridge via ServiceVirtualIPOnGW. This flow is for resolving the ARP requests to ServiceVirtualIPOnGW.

Reply:
Updated.